Updated Database.md

Sukalyan2003 opened the pull request:
<!-- Instructions To Create a Pull Request -->

# Issue Number : <!-- Mention Issue Number Here -->
.
.
.
# Modified File Path : <!-- path of the modified file `eg : ./src/code/main.cpp` -->
.
.
.
# Description : <!-- What have you done/modified/changed -->
. removed a copy of  Gitkraken entry in the table
.
.
# Mentions : <!-- If You Want To Mention any Contributos -->
@ShreyamMaity

<!-- Enjoy. Happy Contributing </> -->

Sukalyan2003 commented:
I removed all the lines i found were duplicates. I think there's a relevant issue about it too.( #52)

I'll remove more if i find other duplicates.

ShreyamMaity commented:
a lot of PRs got merged, check if you're removing a duplicate which has data then remove the other one

Sukalyan2003 commented:
I did that too. I deleted the repo, cloned a fresh copy with the fork synced to latest version of the repo. And there weren't any duplicates i deleted with data. 

The gitkraken one had a link but the other one had a better link for the same so i deleted it, else the other two were empty.
